ThemingCodelab/app/src/main/java/com/example/reply/ui/components/ReplyEmailListItem.kt

```diff
@@ -0,0 +1,107 @@
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.reply.ui.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.StarBorder
+import androidx.compose.material3.Card
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.semantics.selected
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import com.example.reply.data.Email
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ReplyEmailListItem(
+    email: Email,
+    isSelectable: Boolean = false,
+    isSelected: Boolean = false,
+    modifier: Modifier = Modifier,
+    navigateToDetail: (Long) -> Unit
+) {
+    val semanticsModifier =
+        if (isSelectable)
+            modifier
+                .padding(horizontal = 16.dp, vertical = 4.dp)
+                .semantics { selected = isSelected }
+        else modifier.padding(horizontal = 16.dp, vertical = 4.dp)
+    Card(
+        modifier = semanticsModifier.clickable { navigateToDetail(email.id) },
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(20.dp)
+        ) {
+            Row(modifier = Modifier.fillMaxWidth()) {
+                ReplyProfileImage(
+                    drawableResource = email.sender.avatar,
+                    description = email.sender.fullName,
+                )
+                Column(
+                    modifier = Modifier
+                        .weight(1f)
+                        .padding(horizontal = 12.dp, vertical = 4.dp),
+                    verticalArrangement = Arrangement.Center
+                ) {
+                    Text(
+                        text = email.sender.firstName,
+                    )
+                    Text(
+                        text = email.createdAt,
+                    )
+                }
+                IconButton(
+                    onClick = { /*TODO*/ },
```

Comment:
if we're not expecting developers to implement this, replace the comment with something else than todo